src/mawpy/utilities/validations.py

```diff
@@ -0,0 +1,12 @@
+from mawpy.constants import INPUT_ARGS_LIST
+
+
+def check_non_negative(value):
+    if value < 0:
+        raise ValueError(f"{value} is not a valid non-negative numerical value")
+
+
+def validate_input_args(**kwargs):
```

Comment:
```suggestion
def validate_input_args(**kwargs: float) -> None:
```

Reply:
updated for now - but this might change if we add additional validation functions for different types within the parent function.